Application.cpp

```diff
@@ -343,13 +345,16 @@ void Application::buildSwapChain() {
 	glfwGetFramebufferSize(m_window, &width, &height);
 
 	std::cout << "Creating swapchain..." << std::endl;
-	m_swapChainDesc = {};
+	m_swapChainDesc = wgpu::SwapChainDescriptor{};
```

Comment:
I believe this makes no difference (plus, no need to specify `wgpu::` since the namespace is already used).

Reply:
The assignment operator and construtor both take default arguments, therefore the operation was ambiguous